src/main/scala/com/cognite/sdk/scala/v1/fdm/common/properties/PropertyDefinition.scala

```diff
@@ -43,33 +44,64 @@ object PropertyDefinition {
   ) extends ViewPropertyDefinition
       with CorePropertyDefinition
 
-  final case class ConnectionDefinition(
```

Comment:
nice, so there were 2 ConnectionDefinition classes in different packages already :sweat_smile:

Reply:
sounds good to unify as it is in current PR state, so we don't need to split rdr in two

Reply:
Yep, that's what I went for!